admin/broadleaf-open-admin-platform/src/main/resources/open_admin_style/js/admin/lib/plugins/jquery.mCustomScrollbar.js

```diff
@@ -366,6 +366,19 @@ plugin home: http://manos.malihu.gr/jquery-custom-content-scroller
 					$this.mCustomScrollbar("scrollTo",(mCSB_dragger.position().top-(mCSB_draggerDragY))+y,{moveDragger:true,trigger:"internal"});
 				}
 			}
+			function PixelsScrollTo(to){
```

Comment:
Since this is a plugin it's good to add some block comments around these changes to indicate that we changed it and why so that when the plugin gets updated we'll know that we'll likely need to add these changes again